portal-impl/src/com/liferay/portal/service/persistence/impl/RoleFinderImpl.java

```diff
@@ -878,6 +1037,26 @@ protected String getCountByR_U_SQL() {
 		return _countByR_U;
 	}
 
+	protected String getExcludedNames(String[] excludedNames) {
+		int size = excludedNames.length;
+
+		for (String excludedName : excludedNames) {
+			if (excludedName == null) {
```

Comment:
Well, I thought to make a null check in case someone called this method without knowing what they were doing. But it looks like select * from role_ where name not in ('Guest', null, null)" is valid SQL (at least in MySQL) so maybe it isn't necessary.